ide/che-core-ide-app/src/main/resources/org/eclipse/che/ide/CoreLocalizationConstant.properties

```diff
@@ -278,6 +278,7 @@ messages.startingMachine=Starting machine \"{0}\"
 messages.startingOperation=Starting {0}
 messages.machine.not.found=Machine {0} not found
 message.projectCreated=Project {0} created
+message.switch.editors.in.readOnly.mode=Workspace agent is not running, opened editors are in read only mode
```

Comment:
`Workspace agent is not running, open editors are switched to read-only mode`

Reply:
fixed
thank you!